crates/extensions/tedge_config_manager/src/lib.rs

```diff
@@ -78,8 +77,8 @@ impl ConfigManagerBuilder {
             config,
             plugin_config,
             box_builder,
-            download_sender,
-            upload_sender,
+            downloader: download_sender,
+            uploader: upload_sender,
```

Comment:
Minor: side effect of renaming. You can also rename `download_sender` and `upload_sender` here

Reply:
fixed